app/components/works/title_component.html.erb

```diff
@@ -2,7 +2,7 @@
    <header>Title of deposit and contact information</header>
    <div class="mb-3 row">
      <%= form.label :title, 'Title of deposit', class: 'col-sm-2 col-form-label' %>
-     <div class="col-sm-10">
+     <div class="col-sm-9">
```

Comment:
Why was this change made?

Reply:
In order to keep the title input field and the substituent contact email fields the same size. Before the label div was `col-sm-2` and the input div was `col-sm-10`. When a new contact email is added, we needed a `col-sm-1` div for the delete button on the row otherwise the delete button would wrap to the next row because the total number of columns exceeded 12.